jenkins/opensearch-dashboards/integ-test.jenkinsfile

```diff
@@ -14,7 +14,7 @@ lib = library(identifier: 'jenkins@5.11.1', retriever: modernSCM([
 ]))
 
 def docker_images = [
-    "tar": "opensearchstaging/ci-runner:ci-runner-rockylinux8-opensearch-dashboards-integtest-v4",
+    "tar": "opensearchstaging/ci-runner:ci-runner-rockylinux8-opensearch-dashboards-integtest-v4.2",
```

Comment:
I believe this needed to be changed: https://github.com/opensearch-project/opensearch-build/blob/main/manifests/2.11.1/opensearch-dashboards-2.11.1-test.yml#L6?
@peterzhuamazon

Reply:
TBH my code kinda skipped that settings entirely 😄 so it is ok now with this change